app/src/app/utils/helpers.ts

```diff
@@ -119,66 +116,19 @@ export const getFeaturesIntersectingCounties = (
   });
 
   if (!countyFeatures) return;
+  const fips = new Set<string>(countyFeatures.map(c => c.properties.STATEFP + c.properties.COUNTYFP));
```

Comment:
O: Eventually we should create a dedicated column for this so we're not doing this computation so much.